website.py

```diff
@@ -60,4 +60,8 @@ def quick_search(cls):
             (phrase, products.count)
         )
 
-        return render_template('search-results.jinja', products=products)
+        return render_template(
+            'search-results.jinja',
+            products=products,
+            facets=products.result_set.facets
```

Comment:
here you can pass one more arg for all the attributes which have display_count True. so in template you can just check if count need to display or not